tests/Composer/Installers/Test/CakePHPInstallerTest.php

```diff
@@ -83,11 +94,6 @@ public function testGetLocations() {
         $result = $installer->getLocations();
         $this->assertContains('Plugin/', $result['plugin']);
 
-        // special handling for 2.x versions when 3.x is still in development
```

Comment:
This assertion will fail if left in, but I'm pretty sure nobody cares anymore as we are on CakePHP 4. Maybe should find a CakePHP contact to run it by?

Reply:
I believe it's fine. @ADmad can confirm.

Reply:
I don't remember what this special handling was for. CakePHP 4 might be the latest major version but there are still many apps using CakePHP 2.x in production.